### PR TITLE
chore: update links

### PR DESCRIPTION
**Pull request by craicoverflow**
Update links to match the current GitHub org, "redhat-developer"

### CODE REVIEW

**Comment by craicoverflow** [comment]
^^ @pmuir @wtrocki

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -273,7 +273,7 @@ We generate link:https://github.com/redhat-documentation/modular-docs[Modular Do
 
 Every time you submit a pull request a job will run that will verify the generation of modular documentation will succeed.
 
-Every time a pull request is merged to `main`, the modular documentation for the content in the `HEAD` of `main` will be generated to the link:https://github.com/bf2fc6cc711aee1a0c2a/guides/tree/modular-docs[modular docs] branch.
+Every time a pull request is merged to `main`, the modular documentation for the content in the `HEAD` of `main` will be generated to the link:https://github.com/bf2fc6cc711aee1a0c2a/app-services-guides/tree/modular-docs[modular docs] branch.
```
Comment (pmuir) [inline CONTRIBUTING.adoc:276]: Looks like the org needs updating too?

Comment (wtrocki) [reply CONTRIBUTING.adoc:276]: We should probably future proof it by attributes

File: .build/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "description": "Guides for Red Hat OpenShift Streams for Apache Kafka",
   "port": "9001",
-  "repository": "https://github.com/bf2fc6cc711aee1a0c2a/mas-guides.git",
+  "repository": "https://github.com/redhat-developer/app-services-guides.git",
   "homepage": "https://cloud.redhat.com/application-services/openshift-streams",
   "license": "MIT",
```
Comment (craicoverflow) [inline .build/package.json:9]: I should probably change this to Apache 2.0 while I am here?

File: kafkacat/README.adoc [hunk 3]
```diff
@@ -11,11 +11,11 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
 :community:
 :imagesdir: ./images
-:product-long: bf2fc6cc711aee1a0c2a
-:product: bf2
+:product-long: app-services
```
Comment (wtrocki) [inline kafkacat/README.adoc:14]:
```suggestion
:product-long: Application Services
```
Screenshot 2021-04-09 at 14 25 39


Screenshot 2021-04-09 at 14 25 39

Comment (wtrocki) [reply kafkacat/README.adoc:14]: Application services and short ones can be App Services. 
Screenshot 2021-04-09 at 14 27 24

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Minor suggestion added

File: .build/package.json (hunk 2 above)
Comment (wtrocki) [reply .build/package.json:9]: No problem. Cloud mosaic is MIT

**Review by wtrocki — APPROVED** [review]

**Pull request merged**